tests/store/model_registry/test_rest_store.py

```diff
@@ -33,327 +32,353 @@
 from mlflow.utils.rest_utils import MlflowHostCreds
 
 
-@pytest.fixture(scope="class")
-def request_fixture():
-    with mock.patch("requests.request") as request_mock:
-        response = mock.MagicMock()
-        response.status_code = 200
-        response.text = "{}"
-        request_mock.return_value = response
-        yield request_mock
-
-
-def mock_http_request(f):
-    @functools.wraps(f)
-    @mock.patch(
+@contextmanager
+def mock_http_request_200():
+    with mock.patch(
```

Comment:
We can move this to utils module and all tests can use it.

Reply:
Fixed